suites/apis/googleServiceAccountTest.js

```diff
@@ -347,55 +402,56 @@ Scenario('Register Google Service Account for commons project without privilege
 
-// Scenario('Register Google Service Account for commons project
-// where one member does not have privilege @reqGoogle', async (fence, users, google) => {
-//   const googleProject = fence.props.googleProjectA;
-//   const userWithPrivilege = users.mainAcct;
-//   const userWithoutPrivilege = users.auxAcct1;
-// //does NOT have privilege for requested commons proj
-//   const commonsProjectAccessList = ['DEV'];
-//
-//   // Setup
-//   await fence.complete.forceLinkGoogleAcct(
-//     userWithPrivilege,
-//     googleProject.owner,
-//   );
-//   // Add another email to the GCP for us to link to
-//   // Find a user's google email meeting following conditions
-//   //   -an email that is NOT the owner of the google project
-//   //   -an email that is NOT the email of the userWithoutPrivilege
-//   //   -an email that is NOT the email of the userWithPrivilege
-//   // It is assumed that all users' usernames are google account emails
-//   const differentGoogleEmail = Object.values(users).find(user =>
-//     user.username !== googleProject.owner &&
-//     user.username !== userWithoutPrivilege.username &&
-//     user.username !== userWithPrivilege.username,
-//   ).username;
-//   const newRole = {
-//     role: 'roles/viewer',
-//     members: [`user:${differentGoogleEmail}`],
-//   };
-//   await google.updateUserRole(googleProject.id, newRole);
-//   // Link to the email
-//   await fence.complete.forceLinkGoogleAcct(userWithoutPrivilege, differentGoogleEmail);
-//
-//   // Register account for a commons project one user does not have access to
-//   const registerRes = await fence.do.registerGoogleServiceAccount(
-//     userWithPrivilege,
-//     googleProject,
-//     commonsProjectAccessList,
-//   );
-//   fence.ask.responsesEqual(
-//     registerRes,
-//     fence.props.resRegisterServiceAccountMissingProjectPrivilege,
-//   );
-// });
+Scenario('Register SA for data access where one Project member does not have privilege @reqGoogle', async (fence, users, google) => {
+  const googleProject = fence.props.googleProjectA;
+  const userWithPrivilege = users.mainAcct;
+  const userWithoutPrivilege = users.auxAcct1;
+
+  //does NOT have privilege for requested commons proj
+  const commonsProjectAccessList = ['DEV'];
+
+  // Setup
+  await fence.complete.forceLinkGoogleAcct(
+    userWithPrivilege,
+    googleProject.owner,
+  );
+  // Add another email to the GCP for us to link to
+  // Find a user's google email meeting following conditions
+  //   -an email that is NOT the owner of the google project
+  //   -an email that is NOT the email of the userWithoutPrivilege
+  //   -an email that is NOT the email of the userWithPrivilege
+  // It is assumed that all users' usernames are google account emails
+  const differentGoogleEmail = Object.values(users).find(user =>
+    user.username !== googleProject.owner &&
+    user.username !== userWithoutPrivilege.username &&
+    user.username !== userWithPrivilege.username,
+  ).username;
```

Comment:
I wrote this hacky code - I don't really like how I did this previously but currently we don't have something implemented to find this easily. I would just add this extra bit to make it crash and burn if we don't find an email to use.
```suggestion
  ).username;
  if (differentGoogleEmail === undefined) {
    throw "Unable to find a user to add to Google project";
  }
```